.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "env/bin/python3.7"
+    "python.pythonPath": "myvenv/bin/python"
```

Comment:
In this line, is important to verify if your environment is configured correctly with python 3, otherwise your project can not be correctly interpreted in some point due to incompatibility.